src/NuGet.Core/NuGet.Commands/SourcesCommands/SourceRunners.cs

```diff
@@ -317,6 +338,12 @@ public static void EnableOrDisableSource(PackageSourceProvider sourceProvider, s
                 getLogger().LogMinimal(string.Format(CultureInfo.CurrentCulture,
                     Strings.SourcesCommandSourceDisabledSuccessfully, name));
             }
+
+            if (packageSource.IsHttp && !packageSource.IsHttps)
+            {
+                var operation = enable ? "enable" : "disable";
```

Comment:
I think it makes sense to add a warning if customers are trying to `enable` HTTP sources, but I think we shouldn't warn when customers `disable` an HTTP source because the action is secure.

[dotnet-nuget-disable-source](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-disable-source)
[dotnet-nuget-enable-source](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-enable-source)

Reply:
I agree - disabling shouldn't be noisy. In fact, that's a mitigation for the warning.